os/fs/driver/mtd/smart.c

```diff
@@ -493,73 +493,6 @@ static void smart_free(FAR struct smart_struct_s *dev, FAR void *ptr)
 }
 #endif
 
-/***************************************************************************************
- *   This is called from the fs_clean() API to release the memory acquired by MTD driver
- **************************************************************************************/
-int smart_clean(FAR struct inode *inode)
```

Comment:
Ok I will add later, 
But there's no way to free resources which using smartfs, so we added smart_clean.
That's why we added fs_clean to use instead of umount if user wants only unmount not erase flash.
I already gave this comment before, but ignored.
Now if user umount in tash, there might be leakage....